docs/usage/configuration/config_documentation.md

```diff
@@ -3717,7 +3769,7 @@ instance_map:
 
 Experimental: When using workers you can define which workers should
 handle event persistence and typing notifications. Any worker
```

Comment:
I think this is out of date, looks like we have more streams:

https://github.com/matrix-org/synapse/blob/32fc3b7ba4702a0068a82bdd0595e2f426967d4d/synapse/config/workers.py#L72-L113

Reply:
It is out-of-date. See 
* #14197 

for the PR with additional information.

Reply:
I have reword this: https://github.com/matrix-org/synapse/pull/14086/commits/a3ad96aefc518e3faf85de406df36a92f83d7c50
The list of stream writers is currently part of the [worker documentation](https://matrix-org.github.io/synapse/latest/workers.html#stream-writers).

Reply:
Thanks, I think this is great.